src/ScalarMessaging.ts

```diff
@@ -715,6 +715,9 @@ const onMessage = function(event: MessageEvent<any>): void {
         } else if (event.data.action === Action.SetWidget) {
             setWidget(event, null);
             return;
+        } else if (event.data.action === Action.GetOpenIdToken) {
+            getOpenIdToken(event);
+            return;
```

Comment:
the comment on line 709 "Get and set user widgets" seems to be incomplete now. I don't think it adds any value.